database/ffldb/db.go

```diff
@@ -1331,6 +1357,58 @@ func (tx *transaction) FetchBlockHeaders(hashes []chainhash.Hash) ([][]byte, err
 	return headers, nil
 }
 
+// GetBlockCount counts all blocks stored in the database. This function has to
+// iterate a cursor over all blocks and thus is relatively slow.
+//
```

Comment:
The comment on both the interface method and here should call out that it will return `ErrTxClosed`.